src/nfs-ganesha/main.c

```diff
@@ -60,7 +60,7 @@ struct SaunaFSModule SaunaFS = {
                  .acl_support = 0,
 #endif
                  .cansettime = true,
-                 .homogenous = true,
+                 .homogeneous = true,
```

Comment:
This change is modifying a field name inside a structure, so it should fail at build time.

Reply:
is this an external API which should not be changed/fixed without major version boost or internal one? (I found no use of that field in this codebase)

Reply:
is there CI to test the build? (all PRs seems "waiting" for some good old jenkins)

Reply:
> is this an external API which should not be changed/fixed without major version boost or internal one? (I found no use of that field in this codebase)

Yes, this is part of the FSAL API by Ganesha NFS.

> is there CI to test the build? (all PRs seems "waiting" for some good old jenkins)

Currently no, you'll have to build it yourself. I can also run the tests manually if needed and confirm there's no breaking changes (though I doubt there will be)

You can use these commands to build it all (including Ganesha support):
`mkdir build; cd build`
`sudo cmake -DENABLE_TESTS=1 -DENABLE_DOCS=1 -DENABLE_NFS_GANESHA=1 -DENABLE_CLIENT_LIB=ON -DENABLE_URAFT=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo ..`

And then:

`make -j[number of parallel jobs]`

Reply:
added pragma to ignore this line

Automated CI build testing is nearly the first thing I set it up for any new project - wouldn't trust myself or anyone else otherwise these days ;)  If I were to start using saunafs, I would have also added it as a filesystem to test git-annex like we do already for NFS: https://github.com/datalad/git-annex/blob/master/.github/workflows/build-ubuntu.yaml#L281

Reply:
> Automated CI build testing is nearly the first thing I set it up for any new project - wouldn't trust myself or anyone else otherwise these days ;) If I were to start using saunafs, I would have also added it as a filesystem to test git-annex like we do already for NFS: https://github.com/datalad/git-annex/blob/master/.github/workflows/build-ubuntu.yaml#L281

I mean we have it, but we have two problems.
1. The CI is sometimes a bit broken. So we have a rule if the tests pass locally, then it can be merged without CI approval.
2. Currently we don't allow running it and viewing outside the organization, so it's hard for contributors to see what's wrong.

Reply:
If you point me to your setup for the other CI, I should be quickly replicate as a github action -- should then be safe (unless needs secrets) to run it for public PRs etc.

Reply:
I'll run it manually on the Jenkins CI, no worries.

> If you point me to your setup for the other CI, I should be quickly replicate as a github action -- should then be safe (unless needs secrets) to run it for public PRs etc.

This is actually something we are looking into at the moment. I made some internal tool to help me run the tests as quickly as possible for quick iteration. This is not published yet nor do I think it's ready for CI use, but yesterday I talked with @ralcolea and he's also working on something similar as well.

Reply:
I think this conversation is solved. I will close it.